src/service/shamazinService.js

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+
+class ShamazinService{
+    getItemById(id){
```

Comment:
I don't see a problem with this, but for simplicity you may want to consider exporting just the getItemById function when you only have one function. I would think exporting an object is better when you're exporting multiple functions.